osgeo_importer/templates/osgeo_importer/new.html

```diff
@@ -48,9 +48,6 @@ <h2 class="page-title">{% trans "Add data" %}</h2>
                 <i class="fa fa-warning">  {{ error|escape }}</i>
             {% endfor %}
             </div>
-            <div class="col-md-6" style="padding-top: 5px; padding-left: 2px;">
```

Comment:
Disregard my previous comment; forgot https://github.com/sarasafavi/django-osgeo-importer/blob/749c9561168df36cc2ca363bd26911b920909c4d/osgeo_importer_prj/settings.py#L83 was in here now, so I'll update shortly with a fix incorporating that settings var